postgresql_metrics/default_metrics.py

```diff
@@ -143,3 +143,10 @@ def metric_replication_delay_bytes(client_addr, value):
                                  {'what': 'replication-delay-bytes',
                                   'slave': client_addr,
                                   'unit': 'B'})
+
+
+def metric_incoming_replication_running(replication_host, value):
+    return create_default_metric(value == "streaming",
+                                 {'what': 'incoming-replication-running',
+                                  'master': replication_host,
+                                  'unit': 'msg'})
```

Comment:
Did you try how this metric shows up in Alien? Was wondering how it worked with non-numeric values. Cannot check now myself, and could check maybe next week.

Reply:
I've changed the metric to be numeric with the latest PR commit (1 or 0), so now it should be fine in Alien.